src/model/testcases.nit

```diff
@@ -0,0 +1,96 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Add tests and player's submission for missions
+#
+# TODO: only stub right now
+module testcases
+
+import missions
+import players
+import markdown
+import poset
+
+redef class Mission
+	# The set of unit tests used to validate the mission
+	#
+	# `Mission` is refined instead of a subclass to limit the number of classes
+	# and maybe simplify the serialization/API.
+	# If a mission has no test-case, an empty array should be enough for now.
+	var testsuite = new Array[TestCase]
+end
+
+# A single unit test on a mission
+#
+# They are provided by the author of the mission.
+class TestCase
+
+	# Provided input
+	var i: String
```

Comment:
A proposal?

* input/output  -> conflict with kernel::Object::output
* in/out -> `in` is a keyword
* i/o -> no conflict!